src/vars.rs

```diff
@@ -30,12 +30,25 @@ pub struct Vars {
     // multiple allocations and lifetime tracking.
     blank: Var,
     default_recipe_prefix: Var,
+    make_var: Var,
+    makeargs_var: Var,
```

Comment:
I don't think these need to be separate from the `map`, as the variables should be initialized and are allowed to be modified by makefiles. I think you should just insert them into the `map` on iniitalization.

Reply:
Done